ocs_ci/ocs/platform_nodes.py

```diff
@@ -290,3 +304,44 @@ def get_volume_attachments(ebs_volume):
                 f"Volume {volume.id} failed to be attached to an EC2 instance"
             )
             return False
+
+    def restart_nodes_teardown(self):
+        """
+        Make sure all EC2 instances are up. To be used in the test teardown
+
+        """
+        # Get all cluster nodes objects
+        ocp_nodes = get_node_objs()
+
+        # Get the cluster nodes ec2 instances
+        ec2_instances = self.get_ec2_instances(ocp_nodes)
+        assert ec2_instances, (
+            f"Failed to get ec2 instances for node {[n.name for n in ocp_nodes]}"
```

Comment:
If you print list of nodes maybe better to have:  `for nodes: {[....`

Reply:
@petr-balogh thanks for noticing. I'll add more tests in this file soon so will fix that